cli/cron-update-versions.php

```diff
@@ -37,38 +37,38 @@
 
 if (!isset($joomlaReleases->releases))
 {
-    // Exiting here as it seems the API has issues.
-    exit;
+	// Exiting here as it seems the API has issues.
+	die(1);
 }
 
 foreach ($joomlaReleases->releases as $joomlaRelease)
 {
-    /**
-     * object(stdClass)#8 (4) {
-     *   ["version"]=>
-     *   string(6) "3.9.20"
-     *   ["branch"]=>
-     *   string(9) "Joomla! 3"
-     *   ["date"]=>
-     *   string(25) "2020-07-14T15:00:00+00:00"
-     *   ["relationships"]=>
-     *   object(stdClass)#5 (1) {
-     *    ["signatures"]=>
-     *    string(57) "https://downloads.joomla.org/api/v1/signatures/cms/3-9-20"
-     *   }
-     * }
-     */
-
-    // We only want Joomla 3 and 4 Releases here
-    if ($joomlaRelease->branch === 'Joomla! 3' || $joomlaRelease->branch === 'Joomla! 4')
-    {
-        $joomlaJsonData[] = $joomlaRelease->version;
-    }
+	/**
+	 * object(stdClass)#8 (4) {
+	 *   ["version"]=>
+	 *   string(6) "3.9.20"
+	 *   ["branch"]=>
+	 *   string(9) "Joomla! 3"
+	 *   ["date"]=>
+	 *   string(25) "2020-07-14T15:00:00+00:00"
+	 *   ["relationships"]=>
+	 *   object(stdClass)#5 (1) {
+	 *    ["signatures"]=>
+	 *    string(57) "https://downloads.joomla.org/api/v1/signatures/cms/3-9-20"
+	 *   }
+	 * }
+	 */
+
+	// We only want Joomla 3 and 4 Releases here
+	if ($joomlaRelease->branch === 'Joomla! 3' || $joomlaRelease->branch === 'Joomla! 4')
+	{
+		$joomlaJsonData[] = $joomlaRelease->version;
+	}
 }
 
 if (!in_array('4.0.0', $joomlaJsonData))
```

Comment:
This will always be the case for all the Joomla pre-releases. We might want to ping github for the non-stable releases

Reply:
Do we want non stable releases here? We do not have them right now.